connect-location/src/main/java/com/ifttt/location/ConnectLocation.java

```diff
@@ -41,7 +41,7 @@ public void setUpWithConnectButton(ConnectButton connectButton) {
         connectButton.addButtonStateChangeListener(this);
     }
 
-    private ConnectLocation(GeofenceProvider geofenceProvider, ConnectionApiClient connectionApiClient) {
+    ConnectLocation(GeofenceProvider geofenceProvider, ConnectionApiClient connectionApiClient) {
```

Comment:
I don't think we should make this public, just for the tests. 

If absolutely necessary, maybe add a `VisibleForTesting` annotation.

Reply:
This is not public, but package private, it is still not available publicly.

Reply:
sorry I meant package-private. 

If we are changing the visibility from private to package-private only for tests, we can use the `VisibleForTesting` annotation.

Reply:
https://github.com/IFTTT/ConnectSDK-Android/pull/162/commits/8b9fb422a26b3a1ff104a0cab8d033474aa7818d